src/ocamlorg_frontend/pages/home.eml

```diff
@@ -263,22 +263,62 @@ Layout.render
     </div>
   </div>
 </div>
-<div class="bg-legacy-default py-12 overflow-hidden">
-  <div class="container-fluid flex flex-col items-center">
-    <h3 class="font-bold">Releases</h3>
-    <div class="bg-pattern p-6 mt-8 rounded-xl w-full lg:w-2/3 text-white">
-      <a href="<%s Url.release latest_release.version %>">
-        <div class="text-primary text-sm mb-4 uppercase">Recent Release</div>
-        <h3 class="font-bold mb-6">
-          <%s latest_release.version %> <span class="text-sm font-medium">(<%s latest_release.date %>)</span>
-        </h3>
-      </a>
-      <div class="prose prose-sm text-white"><%s! latest_release.highlights_html %></div>
-    </div>
-    <a href="<%s Url.releases %>" class="btn btn-lg btn-ghost mt-8">
-      <span>See all Releases</span>
-      <%s! Icons.chevron_right "h-5 w-5" %>
-    </a>
+<div class="bg-legacy-default pt-24 pb-14 overflow-hidden">
+  <div class="container-fluid flex flex-col md:flex-row gap-10 items-stretch mb-14"> 
+      <div class="basis-1/2">
+        <h3 class="font-bold">Releases</h3>
+        <div class="p-10 mt-7 w-full border-2 shadow-md rounded-2xl h-full">
+          <a href="<%s Url.release latest_release.version %>">
+            <div class="text-primary text-sm mb-4 uppercase">Recent Releases</div>
+            <h3 class="font-bold mb-6">
+              <%s latest_release.version %> <span class="text-sm">(<%s latest_release.date %>)</span>
+            </h3>
+          </a>
+          <div class="prose prose-sm"><%s! latest_release.highlights_html %></div>
+          <a href="<%s Url.releases %>" class="btn btn-lg btn-ghost rounded mt-8">
+            <span>See All Releases</span>
+            <%s! Icons.chevron_right "h-5 w-5" %>
+          </a>
+       </div>
+      </div>
+      <div class="basis-1/2">
+          <h3 class="font-bold">Changelog</h3>
+          <div class="p-10 mt-7 border-2 shadow-md rounded-2xl h-full">
+            <a href="<%s Url.changelog %>">
+            <div class="text-primary text-sm mb-4 uppercase">Releases & Updates</div>
+            <h3 class="font-bold mb-6">
+              Compiler and Platform Tools
+            </h3>
+            </a>
+            <% changelogs |> List.iter (fun (item : Data.Changelog.t) -> %>
+            <div class="pl-8 border-l-2">
+              <div class="flex flex-col lg:flex-row gap-0 lg:gap-16 mb-2 lg:mb-0">
+                <div class="flex flex-col lg:flex-row relative gap-6 lg:gap-10 basis-2/6">
```

Comment:
My impression is that the `basis-2/6` is too small here. Many titles will line-break in a pretty bad way:

![Screenshot 2024-01-03 at 13-27-29 Welcome to a World of OCaml](https://github.com/ocaml/ocaml.org/assets/6594573/ec8a029a-d130-435e-8e69-950d016c7a8a)

The design assumes a container width of 1564px, while in reality we just have a 1280px container here.

We can increase the container's width to 1440px when using `container-fluid wide`, instead of `container-fluid` on the parent div holding the two cards. However, even then, the data doesn't really fit gracefully here.

I think the design needs to be adjusted @Clairevanden.